### ISSUE TITLE
some issues

### ISSUE THREAD

**Issue by GoogleCodeExporter** [andreacessando/blockly#14]
```
blockly publish a aplication in appspot that can be used to program directly on 
the site as a medium blockly.appspot.com develop, in the same place in the 
Export button, js, etc.
that used to blockly's help you can simply create an application for appspot, 
that is, to teach blockly work with google code appspot, that could be used to 
create their own applications
```

Original issue reported on code.google.com by `ikillb...@gmail.com` on 5 Jun 2012 at 7:39

**Comment by GoogleCodeExporter** [comment]
```
referral url: http://habrahabr.ru/post/145067/
```

Original comment by `ikillb...@gmail.com` on 5 Jun 2012 at 7:41

**Comment by GoogleCodeExporter** [comment]
```
If I understand correctly, you'd like the ability to write App Engine 
applications using Blockly and have them uploaded.  This would certainly be a 
great feature.  However at the moment we are very far away from being able to 
do that.  There's no way this could happen for the next year since there is a 
very large list of prerequisites that need to be done first.

I'm going to mark this as "WontFix" but in fact I mean "Will Fix, but a long 
time from now".  Thanks for your suggestion!
```

Original comment by `neil.fra...@gmail.com` on 9 Jun 2012 at 12:05
- Changed state: **WontFix**
- Added labels: **Type-Enhancement**
- Removed labels: **Type-Defect**
